docs/05-concepts/06-database/03-relations/06-modules.md

```diff
@@ -0,0 +1,59 @@
+# Relations with modules
+
+Serverpod [modules](concepts/modules) usually come with predefined tables and data structures. Sometimes it can be useful to extend them with your data structures by creating a relation to the module tables. Relations to modules come with some restrictions since you do not own the definition of the table, you cannot change the table structure of a module table.
+In practice, this means you are never able to create a many-to-many relation as that would require a modification of the module model definition.
+
+## One to one
+
+Creating a relation to a module table is as easy as any other relation, simply reference the module class as the datatype. As for all one-to-one relations, we have to add a unique index to the relation field.
+
+```yaml
+class: User
+table: user
+fields:
+  userInfo: module:auth:UserInfo?, relation
+  age: int
+indexes:
+  user_info_id_unique_idx:
+    fields: userInfoId
+    unique: true
```

Comment:
Wouldn't it be possible to skip the unique index if you would like to create a many to one relation? That is, multiple `Users` connected to one `UserInfo`?

Reply:
Suggest we also include that in the documentation.

Reply:
But then it is no longer a bridge table but a one-to-many relation which you are not able to query with our query system.